sdk/src/main/java/com/oursky/authgear/latte/Latte.kt

```diff
@@ -279,26 +281,38 @@ class Latte(
     fun listenForAppLinks(
         coroutineScope: CoroutineScope,
         callback: suspend (LatteAppLink) -> Unit
+    ) {
+        coroutineScope.launch {
+            intents.collect {
+                val uri = it?.data ?: return@collect
+                val link = LatteAppLink.create(
+                    uri,
+                    appLinkOrigin,
+                    rewriteAppLinkOrigin) ?: return@collect
+                callback(link)
+            }
+        }
+    }
+
+    fun listenForShortLinks(
```

Comment:
I see both listenForShortLinks and listenForAppLinks `collect` `intents`. So the intent will be handled by two subscribers, am I correct?